crates/kitsune_p2p/kitsune_p2p/tests/common/test_legacy_host.rs

```diff
@@ -268,11 +268,20 @@ impl TestLegacyHost {
                                         for interval in intervals {
                                             match interval {
                                                 DhtArcRange::Bounded(lower, upper) => {
-                                                    if lower < op.location()
-                                                        && op.location() < upper
-                                                    {
-                                                        in_any = true;
-                                                        break;
+                                                    if lower <= upper {
```

Comment:
Incomplete test host implementation. The real host already checks the bounded arc this way